packages/mobile/src/components/track-details-tile/TrackDetailsTile.tsx

```diff
@@ -0,0 +1,146 @@
+import type { ID } from '@audius/common'
+import {
+  SquareSizes,
+  getDogEarType,
+  isPremiumContentCollectibleGated,
+  usePremiumContentAccess,
+  cacheUsersSelectors,
+  cacheTracksSelectors
+} from '@audius/common'
+import { View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import IconCollectible from 'app/assets/images/iconCollectible.svg'
+import IconSpecialAccess from 'app/assets/images/iconSpecialAccess.svg'
+import { DogEar, Text } from 'app/components/core'
+import { TrackImage } from 'app/components/image/TrackImage'
+import UserBadges from 'app/components/user-badges'
+import { makeStyles, flexRowCentered, typography } from 'app/styles'
+import { spacing } from 'app/styles/spacing'
+import { useColor } from 'app/utils/theme'
+
+const { getUser } = cacheUsersSelectors
+const { getTrack } = cacheTracksSelectors
+
+const messages = {
+  collectibleGated: 'COLLECTIBLE GATED',
+  specialAccess: 'SPECIAL ACCESS'
+}
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  root: {
+    backgroundColor: palette.neutralLight10,
+    borderWidth: 1,
+    borderColor: palette.neutralLight8,
+    borderRadius: spacing(2),
+    width: '100%'
+  },
+  trackDetails: {
+    ...flexRowCentered(),
+    padding: spacing(4),
+    gap: spacing(4)
+  },
+  trackImage: {
+    width: spacing(22),
+    height: spacing(22),
+    borderRadius: spacing(1),
+    borderColor: palette.neutralLight8
+  },
+  metadataContainer: {
+    gap: spacing(1),
+    flexShrink: 1
+  },
+  premiumContentLabelContainer: {
+    ...flexRowCentered(),
+    gap: spacing(2)
+  },
+  premiumContentLabel: {
+    letterSpacing: spacing(0.5)
+  },
+  trackOwnerContainer: {
+    ...flexRowCentered(),
+    marginTop: spacing(1)
+  },
+  trackOwner: {
+    fontFamily: typography.fontByWeight.medium,
+    fontSize: typography.fontSize.medium
+  }
+}))
+
+type TrackDetailsTileProps = {
+  trackId: ID
+}
+
+export const TrackDetailsTile = ({ trackId }: TrackDetailsTileProps) => {
+  const styles = useStyles()
+  const accentBlue = useColor('accentBlue')
```

Comment:
This pattern feels strange since we pass color names to everything else. But I think I see why we do it. Worth exploring later if we can wrap Icons in a component that lets us pass color names to it.

Reply:
fair